src/engine/Operation.cpp

```diff
@@ -146,12 +142,12 @@ shared_ptr<const ResultTable> Operation::getResult(bool isRoot,
         // Note: both of the following calls have no effect and negligible
         // runtime if neither a LIMIT nor an OFFSET were specified.
         result.applyLimitOffset(_limit);
-        _runtimeInfo.addLimitOffsetRow(_limit, limitTimer.msecs(), true);
+        _runtimeInfo->addLimitOffsetRow(_limit, limitTimer.msecs(), true);
```

Comment:
maybe consistently use `runtimeInfo().addLimitOffsetRow()` etc, and make the `_runtimeInfo` object private.
(also no real need to call the getter `getRuntimeInfo()` as it doesn't really perform work.